test_nested_virtualization.py

```diff
@@ -0,0 +1,50 @@
+"""
+
+Nested Virtualization Acceptance Tests
+====================
+
+Customers can start virtual servers within Plus and Flex servers.
+
+"""
+
+
+def test_virtualization_support(create_server):
+    """ Nested virtualization is supported. """
+
+    # List of virt-host-validate checks that do NOT need to "PASS"
+    virt_validate_pass_exceptions = {
+        "Checking for device assignment IOMMU support",
+        "Checking for secure guest support",
+        "Checking for cgroup 'freezer' controller support",
+    }
+
+    # Start a server with the flex-4-1 flavor
+    server = create_server(flavor='flex-4-1')
+
+    virt_validate_status = server.prepare_nested_virtualization()
+
+    # Validate all checks PASS except for the ones defined above
+    for line in virt_validate_status.splitlines():
+        parts = line.split(':')
+        description = parts[1].strip()
+        status = parts[2].strip().split()[0]
+
+        if description not in virt_validate_pass_exceptions:
+            assert status == "PASS"
+
+
+def test_run_nested_vm(create_server):
+    """ Nested virtualization is supported. """
+
+    vm_os = 'alpinelinux3.17'   # Needs to match one virt os-variant
+    vm_iso_url = (
```

Comment:
We should not use externally hosted ISOs for test. They cost bandwidth and causes our tests to fail if there's an issue with someone else's CDN.

We already store custom alpine images here: https://at-images.objects.lpg.cloudscale.ch/ (search for it in the code), I think we can add an ISO there, making the URL much shorter in the process.

Even easier might be to just download our custom alpine image, and run that. If I'm not missing something you are only downloading the ISO anyway, not actually mounting it or anything.